examples/substrate-react-example/src/substrate-lib/SubstrateContext.tsx

```diff
@@ -0,0 +1,97 @@
+/* eslint-disable */
```

Comment:
since we are not allowing this under our code, we should use eslint-disable over the project